x-pack/plugins/actions/README.md

```diff
@@ -234,9 +234,24 @@ Payload:
 
 ## Firing actions
 
-The plugin exposes an execute function that you can use to run actions.
+Running actions is possible by using the ActionsClient which is provided by the plugin's Start Contract.
```

Comment:
nit: to mention they have to use `getActionsClientWithRequest` function to get the actions client.
```suggestion
Running actions is possible by using the ActionsClient which is provided by the `getActionsClientWithRequest` function part of the plugin's Start Contract.
```